### PR TITLE
Feat: Add Clone/Transform workflows

### PR DESCRIPTION
**Pull request by arnaudon**

### CODE REVIEW

**Comment by arnaudon** [comment]
allen sdk install is broken due to compile of statsmodel, so i discarded so it can install

**Comment by arnaudon** [comment]
and neuroc0.3.0 exists...

File: setup.py [hunk 1]
```diff
@@ -69,7 +73,7 @@
     extras_require={
         "docs": doc_reqs,
         "test": test_reqs,
-        "allen_brain": ["allensdk>=2.13.5"],
+        #"allen_brain": ["allensdk>=2.13.5"],
```
Comment (adrien-berchet) [inline setup.py:76]: I think we can just get rid of AllenSDK thanks to https://github.com/brainglobe/morphapi/pull/69, we just have release a new version of MorphAPI and then update the min version here.

**Comment by arnaudon** [comment]
it seems unable to write some files... I don't get why..

**Comment by adrien-berchet** [comment]
> it seems unable to write some files... I don't get why..

For the records, this is the fix: https://github.com/h5py/h5py/issues/1101

**Comment by arnaudon** [comment]
waiting on https://github.com/BlueBrain/morph-tool/pull/132

**Pull request merged**